Ginger/GingerCore/Actions/ActScript.cs

```diff
@@ -141,10 +141,10 @@ protected void AddData(string outLine)
         }
         public override void Execute()
         {
-            if (ScriptName == null)
+            if ((ScriptName == null) && (GetInputParamCalculatedValue("Free Command") == null || GetInputParamCalculatedValue("Free Command") == string.Empty)) 
```

Comment:
you can directly check string.IsNullorEmpty for input param free command instead of 2 conditions

Reply:
Please confirm if you are handling this